app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user
+
+  def create
+    @club = Club.find(params[:club_id])
+    @comment = Comment.new(comment_params)
+    @comment.club = @club
+    @comment.user = current_user
+    if @comment.save
+      redirect_to @club, notice: "Comment Saved"
+    else
+      @membership = Membership.find_by(user_id: current_user.id,
+                                       club_id: @club.id)
+      render 'clubs/show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.